README.md

```diff
@@ -227,7 +227,7 @@ const data = {
 
 #### `response.body`
 
-Nodejs has two kinds of streams: [web streams](https://nodejs.org/dist/latest-v16.x/docs/api/webstreams.html) which follow the API of the WHATWG web standard found in browsers, and an older Node-specific [streams API](https://nodejs.org/api/stream.html). `response.body` returns a readable web stream. If you would prefer to work with a Node stream you can convert a web stream using `.fromWeb()`.
+Nodejs has two kinds of streams: [web streams](https://nodejs.org/dist/latest-v16.x/docs/api/webstreams.html), which follow the API of the WHATWG web standard found in browsers, and an older Node-specific [streams API](https://nodejs.org/api/stream.html). `response.body` returns a readable web stream. If you would prefer to work with a Node stream you can convert a web stream using `.fromWeb()`.
```

Comment:
What's changed?

Reply:
Thrown the comma in front of "which" because it is introducing a non-defining/non-restrictive clause.

I think the following two articles explain this change, and the following changes you've flagged, far better than I can!:

- [Grammarly - Which vs That](https://www.grammarly.com/blog/which-vs-that/)
- [Grammarly - Comma Before Which](https://www.grammarly.com/blog/comma-before-which/ )